terraform/webserver/templates/network_security_group.tf

```diff
@@ -1,33 +1,39 @@
-resource "azurerm_network_security_group" "kunlun_network_security_group" {
-  name                = "${var.env_name}-nsg"
+resource "azurerm_network_security_group" "kunlun_jumpbox_network_security_group" {
```

Comment:
My opinion is to include nsg in jumpbox.tf and web_server.tf. We don't need a separated tf for nsg.

Reply:
It makes sense. I'll change the code.